### PR TITLE
Partial resolution Issue 736

### PR DESCRIPTION
**Pull request by ghost**
First pass removal of conditional 'non-skip' blind returns in test cases.

Addressing: https://github.com/dotnet/sdk/issues/736

### CODE REVIEW

File: src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToStoreAProjectWithDependencies.cs [hunk 1]
```diff
@@ -283,15 +283,16 @@ public void It_creates_profiling_symbols()
...
-            var newtonsoftSymbolsFolder = symbolsFolder.Sub("newtonsoft.json").Sub("9.0.1").Sub("lib").Sub("netstandard1.0");
-            newtonsoftSymbolsFolder.Should().Exist();
-
-            var newtonsoftSymbolsFiles = newtonsoftSymbolsFolder.GetFiles().ToArray();
-            newtonsoftSymbolsFiles.Length.Should().Be(1);
-            newtonsoftSymbolsFiles[0].Name.Should().StartWith("Newtonsoft.Json").And.EndWith(symbolFileExtension);
+                var newtonsoftSymbolsFiles = newtonsoftSymbolsFolder.GetFiles().ToArray();
+                newtonsoftSymbolsFiles.Length.Should().Be(1);
+                newtonsoftSymbolsFiles[0].Name.Should().StartWith("Newtonsoft.Json").And.EndWith(symbolFileExtension);
+            }
```
Comment (nguerrera) [inline src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToStoreAProjectWithDependencies.cs]: Why this change?

Comment (johnbeisner) [reply src/Tests/Microsoft.NET.Publish.Tests/GivenThatWeWantToStoreAProjectWithDependencies.cs]: There is a blind "return;" 
The test should execute block A or block B based on the conditional "RuntimeInformation.IsOSPlatform(OSPlatform.OSX)"- essentially the return is acting as the 'else'; therefore, I added it.

**Comment by johnbeisner** [comment]
@dotnet/dotnet-cli 
Can I get an approval if appropriate.

**Comment by johnbeisner** [comment]
Replaced by: https://github.com/dotnet/sdk/pull/1872
